README.md

```diff
@@ -63,7 +63,9 @@ extension=json.so
 
 Extensions can be built using the lambci/lambda:build-nodejs8.10 Docker image. It is recommended that custom extensions be provided by a separate Lambda Layer with the extension .so files placed in /lib/php/7.1/modules/ so they can be loaded alongside the built-in extensions listed above.
 
-#### Example
+The extensions can be loaded by adding an ini file in the etc/php-7.1.d this will be picked up automatically
```

Comment:
I'm going to edit this a bit post-merge to make it clearer for both layer and package code.